### PR TITLE
Implement membership opt in and out operations in OOTB

### PR DESCRIPTION
**Pull request by gitCarrot**
# Ticket Link

[Ticket-1662](https://uhawaii.atlassian.net/browse/GROUPINGS-1662)

# List of squashed commits

- Use query functions from ootbPropertiesService to return the results wrapper object with single line

- Create getter function for accessing ws bean in wrapper class

- Add the tests for opt in and opt out

- Modify the data harness

- Modify the tests for opt in and opt out

- Modify execution logic in findGroupResults test with new findGroup function

- Fix add, remove member(s) test by adding getGroupResults mock

- Clean up the code

- Get back from OOTB to GROUPER service type
- Fix codacy code analysis
- Back to previous code style on the application.properties

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [x] Project Integration Tests Passed:
- [x] Executes Expected Functionality:
- [x] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

File: src/main/resources/application.properties
Comment (JorWo) [inline src/main/resources/application.properties:1]: I'm curious if our codestyle auto-formatted this. I would undo the changes in this file because the spacing makes the properties file more organized.

**Review by mhodgesatuh — APPROVED** [review]

**Comment by mhodgesatuh** [comment]
I approved, but Jordan's final review is key to a successful merge.

**Review by JorWo — APPROVED** [review]
